Settings/ViewControllers/BKPubKey/BKPubKeyViewController.m

```diff
@@ -299,4 +349,30 @@ - (void)showKeyInfo:(NSIndexPath *)indexPath
   [self performSegueWithIdentifier:@"keyInfoSegue" sender:self];
 }
 
+#pragma mark - UIDocumentPickerDelegate
+
+- (void)documentPickerWasCancelled:(UIDocumentPickerViewController *)controller {
+  
+  
+}
+
+- (void)documentPicker:(UIDocumentPickerViewController *)controller didPickDocumentsAtURLs:(NSArray<NSURL *> *)urls {
+  
+  NSArray *readFileUrls = urls;
+  
+  for (NSURL *fileUrl in readFileUrls) {
+    
+    NSString *keyString = [NSString stringWithContentsOfURL:fileUrl encoding:NSUTF8StringEncoding error:NULL];
```

Comment:
We can call `documentPicker:didPickDocumentAtURL:` here to avoid duplication.

Reply:
Removed implementation of `documentPicker:didPickDocumentAtURL:` it's deprecated. We're only gonna use `documentPicker:didPickDocumentsAtURLs:`.